src/components/SortHeader/SortHeader.tsx

```diff
@@ -0,0 +1,21 @@
+import { SortType } from '../../types/SortType';
+import { SortLink } from '../SortLink/SortLink';
+
+export const SortHeader = () => {
+  return (
+    <tr>
+      {Object.values(SortType).map(value => (
+        <th key={value}>
+          <span className="is-flex is-flex-wrap-nowrap">
+            {`${value.slice(0, 1).toUpperCase()}${value.slice(1)}`}
```

Comment:
It's better to do such transformation using a css rule `text-transform: capitalize;`. But this css will make only the first letter capitalized, so it will not work if the text is uppercased, but in your example, it'll be fine.